.github/workflows/ci.yaml

```diff
@@ -52,8 +52,11 @@ jobs:
       - name: Scaladoc
         run: sbt ++$SCALA_VERSION doc
   coverage:
-    needs: [validate]
+    needs: validate
     name: Test Coverage
+    if: |
+      github.ref == 'refs/heads/master' && (needs.check-duplicate.result == 'success' || needs.check-duplicate.result == 'skipped') ||
+      github.event_name == 'pull_request' && needs.check-duplicate.result == 'success'
```

Comment:
Ideally this is what we want, but codeclimate is blocking the pipeline because it was (rightly) skipped. Let's allow test coverage to run on success or skip for PRs for now, and then a possible fix is to provide a manual way of firing the actions overriding the skip.